.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+# forces black to format code before every commit d
+repos:
+  - repo: https://github.com/psf/black
+    rev: 23.3.0
+    hooks:
+      - id: black
+        language_version: python3.9
```

Comment:
Can we not use at least `python3.10` as that is what the repo required from the `pyproject.toml`